am/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: am
+version: 1.0.0
```

Comment:
i would start from a <1.0.0 and move to 1.0.0 when it is tested enough

Reply:
Fixed